integration/tests/cook/util.py

```diff
@@ -538,11 +553,15 @@ def submit_jobs(cook_url, job_specs, clones=1, pool=None, headers=None, log_requ
         headers = {}
     if isinstance(job_specs, dict):
         job_specs = [job_specs] * clones
-
+    caller = get_caller()
     def full_spec(spec):
+        if 'name' not in spec:
+            spec['name']="default_job-"+caller
```

Comment:
Why are we using `default_job-` here but `default_test_job-` above?

Reply:
Made consistent.